### PR TITLE
Add a section on cross-account access for an S3 bucket

### PR DESCRIPTION
**Pull request by sihil**

### CODE REVIEW

**Comment by steppenwells** [comment]
looks god :+1:

**Comment by adamnfish** [comment]
Nice one.

File: README.md [hunk 1]
```diff
@@ -246,12 +246,37 @@ Access to the s3 bucket is controlled by overriding the `awsCredentials` and `aw
...
+Add something like the following statement to the bucket policy (if there is no existing policy then you'll need the skeleton in place as this fragment is only
+a statement).
+
+```
+{
+  "Sid": "gu-aws-composer-account",
+  "Effect": "Allow",
+  "Principal": {
+    "AWS": "arn:aws:iam::743583969668:root"
+  },
```
Comment (lindseydew) [inline README.md]: +1 this approach is much nicer than passing aws keys around. 

Just wondering, does this give read access to all users of the account the private file?

**Comment by philwills** [comment]
Do we also need to change https://github.com/guardian/pan-domain-authentication/blob/master/pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/service/S3Bucket.scala#L18, so that we can do things like exploit IAM instance profiles to access this information sanely?

Also, this PR has been open for a long time, is there any reason not to merge? Is anyone using this?

**Comment by sihil** [comment]
Closing as superseded by #21

**Pull request closed without merging**